gnovm/pkg/gnolang/gno_test.go

```diff
@@ -17,6 +17,29 @@ import (
 	"github.com/stretchr/testify/require"
 )
 
+func setupMachine(numValues, numStmts, numExprs, numBlocks, numFrames, numExceptions int) *Machine {
```

Comment:
Can you please pass in `*testing.B`, and call `b.Helper()`?

I'm not sure how this wasn't caught by the linter